src/components/ImageView/index.native.js

```diff
@@ -31,6 +32,28 @@ class ImageView extends PureComponent {
         this.doubleClickInterval = 175;
         this.imageZoomScale = 1;
         this.lastClickTime = 0;
+        this.amountOfTouches = 0;
+
+        // PanResponder used to capture how many touches are active on the attachment image
+        this.panResponder = PanResponder.create({
+            onStartShouldSetPanResponder: this.handleOnStartShouldSetPanResponder.bind(this),
+        });
+    }
+
+    /**
+     * Handler for the initial tap for the PanResponder on our ImageZoom overlay View
+     *
+     * @param {Event} e
+     * @param {GestureState} gestureState
+     * @returns {Boolean}
+     */
+    handleOnStartShouldSetPanResponder(e, gestureState) {
```

Comment:
Style guide rule about this [here](https://github.com/Expensify/App/blob/main/STYLE.md#naming-conventions)